### DISCUSSION TITLE
Opening a large database in SQLiteStudio takes long time

### DISCUSSION THREAD

**Discussion by mtirfan66** [pawelsalawa/sqlitestudio#4978] [Q&A]
Hi,
The sqlitestudio works great with me. I have the latest version 3.4.4. I have two issues which I wanted to check with everyone to see if   I am doing everything right.  The first one, described below, is on opening a large database in the studio. The second one is on copying a table from one database to another.
Here are the details of the first problem.
I have a large   data   file - about 370 mb in size with over 5000 tables each with 200 rows. Each table has a primary key, they are not related to each other though. The file gets updated roughly one to two times each month. 
I need to look at the data frequently though, by opening the file in the studio and opening the tables that I need to look into.  
It takes quite a bit of time each time I open this database in sqlitestudio by double clicking on the .db file.
Is there something I can do to the database (other than splitting it) to accelerate the loading of it in the sqlitestudio? Is it possible to load it once and access it later without having to reload even though the sqlitestudio (and, the device) is closed/shutdown in between?
Thanks, in advance.
-Irfan

**Comment by pawelsalawa** [comment]
It may be hard to reproduce your specific conditions. Do you think you could share the database file with me privately (no need to make it public)? You could delete all data from tables. I'm interested mostly in the schema of the database.

**Comment by mtirfan66** [comment]
Hi,I don't have a problem sharing the database. Is this the right email to
share the link to the uploaded file path? Is Google drive okay, or you
would prefer me to drop it some place else?
Thank you.
-Irfan

**Reply by pawelsalawa** [reply]
I'm not sure which email did you mean, but you can use official contact email: [support@sqlitestudio.pl](mailto:support@sqlitestudio.pl) - and yes, google drive is okay.

**Reply by mtirfan66** [reply]
Thanks. Just sent the databases.

**Reply by pawelsalawa** [reply]
Thanks. I've got the files. This needs optimizations. I've created #4980 to track it.

**Reply by pawelsalawa** [reply]
Just so you know, the referenced issue has just been resolved and the fix will be included in 3.4.5 version.
